test/resolvers.spec.js

```diff
@@ -0,0 +1,107 @@
+/* eslint-env mocha */
+'use strict'
+
+const { expect } = require('aegir/utils/chai')
+const sinon = require('sinon')
+
+const multiaddr = require('../src')
+const resolvers = require('../src/resolvers')
+const { Resolver } = require('../src/resolvers/dns')
+
+const dnsaddrStub1 = [
+  ['dnsaddr=/dnsaddr/ams-1.bootstrap.libp2p.io/p2p/QmSoLer265NRgSp2LA3dPaeykiS1J6DifTC88f5uVQKNAd'],
+  ['dnsaddr=/dnsaddr/ams-2.bootstrap.libp2p.io/p2p/QmbLHAnMoJPWSCR5Zhtx6BHJX9KiKNN6tpvbUcqanj75Nb'],
+  ['dnsaddr=/dnsaddr/lon-1.bootstrap.libp2p.io/p2p/QmSoLMeWqB7YGVLJN3pNLQpmmEk35v6wYtsMGLzSr5QBU3'],
+  ['dnsaddr=/dnsaddr/nrt-1.bootstrap.libp2p.io/p2p/QmcZf59bWwK5XFi76CZX8cbJ4BhTzzA3gU1ZjYZcYW3dwt'],
+  ['dnsaddr=/dnsaddr/nyc-1.bootstrap.libp2p.io/p2p/QmSoLueR4xBeUbY9WZ9xGUUxunbKWcrNFTDAadQJmocnWm'],
+  ['dnsaddr=/dnsaddr/sfo-2.bootstrap.libp2p.io/p2p/QmSoLnSGccFuZQJzRadHn95W2CrSFmZuTdDWP8HXaHca9z']
+]
```

Comment:
This is odd, why is this a nested array?

Reply:
True, but this is for compliance with Node.js `dns.promises` API. If you check the table below [here](https://nodejs.org/dist/latest-v14.x/docs/api/dns.html#dns_dnspromises_resolve_hostname_rrtype), the TXT resolver returns [`Array<Array<string>>`](https://nodejs.org/dist/latest-v14.x/docs/api/dns.html#dns_dnspromises_resolvetxt_hostname)